front/app/hooks/useInputSchema.ts

```diff
@@ -47,238 +180,3 @@ export default (projectId: string | undefined) => {
 
   return { schema, uiSchema, inputSchemaError: isError };
 };
-//
-//   if (isNilOrError(authUser)) {
-//     return {};
-//   }
```

Comment:
There was a long piece of commented out code here - I've just decided to remove it instead.